drafts/str4d-stratum/draft1.rst

```diff
@@ -0,0 +1,256 @@
+Abstract
+========
+
+This ZIP describes the Zcash variant of the Stratum protocol, used by miners to
+communicate with mining pool servers.
+
+
+Specification
+=============
+
+``mining.subscribe()``
+----------------------
+
+Request::
+
+    {"id": 1, "method": "mining.subscribe", "params": ["CONNECT_HOST", CONNECT_PORT, "MINER_USER_AGENT", "SESSION_ID"]}\n
+
+- ``CONNECT_HOST`` (str): The host string used by the miner.
+
+  - ``pool.example.com``
```

Comment:
Security consideration: the server should not trust the host string for anything.